TDTChocolate/FoundationAdditions/NSString+TDTAdditions.m

```diff
@@ -13,6 +13,10 @@ + (NSString *)tdt_stringWithUnsignedInteger:(NSUInteger)unsignedInteger {
   return [self stringWithFormat:@"%llu", (unsigned long long)unsignedInteger];
 }
 
++ (NSString *)tdt_stringWithUnsignedLongLong:(uint64_t)unsignedLongLong {
```

Comment:
The method names say "unsigned long long". The implementations works for "unsigned long long". Shouldn't the type also be "unsigned long long" instead of "uint64_t"?

If you want methods for "uint64_t", then the names of the methods should say so.